scripts/CEE/drain-reboot-worker-node/metadata.yaml

```diff
@@ -31,15 +31,18 @@ rbac:
         - ""
         resources:
         - "nodes"
+        - "nodes/proxy"
```

Comment:
Could you help me to understand why this file is also changed? It seems to be in a different folder than the other 3 files.

Reply:
Yep, although this also could apply here, we can disregard this change as it was by mistake.